install_config/aggregate_logging.adoc

```diff
@@ -1518,3 +1518,627 @@ $ oc label node --all logging-infra-fluentd=true <1>
+is a subfield of labels field. Each label name is de-dotted, meaning dots in the
+name are replaced with underscores.
+
+[cols="3,7",options="header"]
+|===
+|Parameter
+|Description
+
+| `kubernetes.pod_id`
+|Kubernetes ID of the pod.
+
+| `kubernetes.namespace_name`
+|The name of the namespace in Kubernetes.
+
+| `kubernetes.namespace_id`
+|ID of the namespace in Kubernetes.
+
+| `kubernetes.host`
+|Kubernetes node name.
+
+| `kubernetes.container_name`
+|The name of the container in Kubernetes.
+
+| `kubernetes.labels.deployment`
+||The deployment associated with the Kubernetes object.
+
+| `kubernetes.labels.deploymentconfig`
+|The deploymentconfig associated with the Kubernetes object.
+
+| `kubernetes.labels.component`
+|The component associated with the Kubernetes object.
+
+| `kubernetes.labels.provider`
+|The provider associated with the Kubernetes object.
+|===
+
+[discrete]
+[[exported-fields-kubernetes.annotations]]
+=== `kubernetes.annotations` Fields
+
+Annotations associated with the OpenShift object are `kubernetes.annotations`
+fields.
+
+[discrete]
+[[exported-fields-docker]]
+=== Docker Fields
+
+Namespace for docker container-specific metadata. The `docker.container_id` is
+the Docker container ID.
+
+[discrete]
+[[exported-fields-pipeline_metadata]]
+=== `pipeline_metadata` Fields
+
+This includes metadata related to ViaQ log collection pipeline. Everything
+related to log collector, normalizers, and mappings goes here. Data in this
+subgroup is stored for troublehsooting purposes mostly. The
+`pipeline_metadata.@version` field is the version of `com.redhat.viaq` mapping
+the document is intended to adhere by the normalizer. It must be set by the
+normalizer. The value must correspond to the [_meta][version]. For example,
+`class` with the description `TODO`, and `region` with the description region
+mapping.
+
+[discrete]
+[[exported-fields-pipeline_metadata.collector]]
+=== `pipeline_metadata.collector` Fields
+
+This section contains metadata specific to the collector.
+
+[cols="3,7",options="header"]
+|===
+|Parameter
+|Description
+
+| `pipeline_metadata.collector.hostname`
+|FQDN of the collector. It might be different from the FQDN of the actual emitter
+of the logs.
+
+| `pipeline_metadata.collector.name`
+|Name of the collector.
+
+| `pipeline_metadata.collector.version`
+|Version of the collector.
+
+| `pipeline_metadata.collector.ipaddr4`
+|IP address v4 of the collector server, can be an array.
+
+| `pipeline_metadata.collector.ipaddr6`
+|IP address v6 of the collector server, can be an array.
+
+| `pipeline_metadata.collector.inputname`
+|How the log message was received on the collector whether it was TCP/UDP, or
+imjournal/imfile.
+
+| `pipeline_metadata.collector.received_at`
+|Time when the message was received at the collector.
+
+| `pipeline_metadata.collector.original_raw_message`
+|The original non-parsed log message, collected by the collector or as close to the
+source as possible.
+|===
```

Comment:
These three entries use different constructs. Is this intentional:
received on the collector
received at the collector
received by the collector